GetDKAN/dkan_sci_metadata
# 1538-fundred-facet: Adding fundref facet for paragraphs

## QA
- Deploy branch
- Tag dataset item with tid from fundref paragraph
- Visit search page, and verify Funding Source facet appears
- Select the term used in step 2 dataset
- Confirm only this dataset appears in search results

Connects isdapps/adc-code/issues/1538
